.github/workflows/build.yml

```diff
@@ -70,40 +92,4 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: ${{ matrix.artifact_name }}
-          path: zed/target/release/zed.exe
-
-  release:
-    needs: build
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main'
-    permissions:
-      contents: write
```

Comment:
Release related steps move to a reusable workflow: [release.yml](https://github.com/deevus/zed-windows-builds/pull/36/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34). Reused by [stable.yml](https://github.com/deevus/zed-windows-builds/pull/36/files#diff-eddaa43b142a4a5fbb9297f1e7b3c59401c1f22790df62f04954baa4370e72e9) and [nightly.yml](https://github.com/deevus/zed-windows-builds/pull/36/files#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6)